src/open_samus_returns_rando/files/guilib.lua

```diff
@@ -78,4 +78,15 @@ if GUILib == nil then
         dna:ForceRedraw()
     end
   end
+
+  function GUILib.UpdateTotalDNAColor()
+    if Game.GetItemAmount(Game.GetPlayerName(), "ITEM_ADN") < 39 then
+      local dnaCounter = GUI.GetDisplayObject("IngameMenuRoot.IngameMenuComposition.LowerComposition.LowerInfoComposition.DNACounter")
+        GUI.SetProperties(dnaCounter, {
+          ColorR = "0.68000",
+          ColorG = "0.83000",
+          ColorB = "0.93000",
+        })
```

Comment:
This is indented too much.
(I serioulsy read the code wrong at first. Phone in vertical mode, which adds additional line wraps and it made it look that this is part of the function call above the line)